project-metrics/metrics_service/models.py

```diff
@@ -1,14 +1,31 @@
 """Defines ORM models for interactions with the DB."""
 
-from typing import Text
+from typing import Text, Type, TypeVar
 import datetime
 import enum
 import sqlalchemy
 from sqlalchemy.ext import declarative
+import timedelta
 
 Base = declarative.declarative_base()
 
 
+def is_last_n_days(
+  timestamp_column: sqlalchemy.orm.attributes.InstrumentedAttribute, days: int
+  ) -> sqlalchemy.sql.elements.BinaryExpression:
+  """Produces the filter expression for the last N days of a model.
+
+  Args:
+    timestamp_column: model column to filter by.
+    days: number of days back to filter by.
+
+  Returns:
+    A binary expression which can be passed to a query's `.filter()` method.
+  """
+  n_days_ago = datetime.datetime.now() - timedelta.Timedelta(days=days)
```

Comment:
🤔 timedelta is built into Python?
`datetime.datetime.now() - datetime.timedelta(days=days)` should achieve the same effect?

Reply:
Didn't know `timedelta` was in `datetime`! Awesome! Will remove the dependency and use the native one when I replicate this code in the upcoming date filtering PR.